### ISSUE TITLE
Latest v Start not showing in LBS

### ISSUE THREAD

**Issue by sbservices** [alicolville/Weight-Tracker#601]
Extension of Issue #592 

Item number 2 here is showing in St/Lbs even though settings are set to 'Pounds' in admin preferences and user preferences.
So when the admin has set the preferences to 'Pounds', you'd expect to be able to show all boxes where applicable in either LBS or ST/LBS. Else by specifying in the shortcode as a preference so it could be tailored in the kiosk if the Coach or admin user preferred to see the weights differently than what is in the user preference settings.  That way the kiosk setting could override the user setting.

<img width="955" alt="image" src="https://github.com/alicolville/Weight-Tracker/assets/101115016/ca484ef2-1991-4fb8-aefa-d19118bb4d24">

Being able to see weight lost since start in LBS is useful to be able to say to the member as you are weighing them in - "ooh Mary, you've only 3 more LBS to get your 50LB award!!"

**Comment by alicolville** [comment]
The units should switched to the settings set by the current logged in user. So check what your weight units are set to when logged in as yourself.

I'll add the bug into the Kan ban ;)

**Comment by sbservices** [comment]
ahh that would be something I would never have thought of! Ok, will check.

**Comment by sbservices** [comment]
I have amended the second box that says 'latest weight' to 'latest-weight-difference-as-weight' for now, so she has the information she needs there. But does this mean that this first box is redundant now and that the info is combined in to the 2nd box?

**Comment by alicolville** [comment]
Yeah, definitely a bug here... think I've found the problem. Does this look better:

<img width="357" alt="image" src="https://github.com/user-attachments/assets/ab1b37cb-3a2b-4add-8802-9501c6d085a8">

**Comment by sbservices** [comment]
Thanks Ali, I'll do the updates when I get a chance. Perhaps over the weekend or something. Thanks a million.
